spec/acceptance/workflow_regen_after_expire_spec.rb

```diff
@@ -3,7 +3,7 @@
 
 # https://forge.puppet.com/puppetlabs/certregen#revive-a-ca-thats-already-expired
 describe "C99821 - workflow - regen CA after it expires" do
-  if hosts_with_role(hosts, 'master').length>0 and hosts_with_role(hosts, 'database').length>0 then
+  if find_install_type == 'pe' then
```

Comment:
1. The 'pe' install type triggers a PE installation which defaults to a monolithic master install (including master, and database). See: https://github.com/puppetlabs/beaker-puppet_install_helper/blob/master/lib/beaker/puppet_install_helper.rb#L47-L50 and https://github.com/puppetlabs/beaker-pe/blob/master/lib/beaker-pe/install/pe_utils.rb
2. This would prevent the FOSS tests from exercising this code. Such an exercise would not pass since the puppetdb query in the gem is specific to PE. See: https://github.com/puppetlabs/puppetlabs-certregen/blob/master/lib/puppet_x/certregen/ca.rb#L114